README.md

```diff
@@ -1,17 +1,27 @@
 # byu-jwt-python
-A python JWT validator that does all the BYU specific stuff as well.
+A python JWT validator that does all the BYU specific stuff as well as handle caching well-known and cert fetching
 
 # Installation
 `pip install byu_jwt`
 
 ## API
 
-### How to Validate a JWT
+Instantiate the class and reuse the object to utilize caching
 
-```
+```python
 import byu_jwt
+byujwt = byu_jwt.JWT()
```

Comment:
I feel like we may want to consider a different class name here. Syntactically this line looks to me like we are creating a JWT, when we are actually creating a kinda nebulous decoder/checker thing. Changing the class name may make that more readable/understandable.